### PR TITLE
Merge changes from 1.4-rc0 back into master

### PR DESCRIPTION
**Pull request by case540**
These contains mostly release note additions and some version string updates.

### CODE REVIEW

**Comment by case540** [comment]
Did we want to bring the ghost batch norm revert back into master jhseu?

**Comment by av8ramit** [comment]
I think we need to remove @jhseu's change from this. Master is correct, we just didn't want that in 1.4. 

https://github.com/tensorflow/tensorflow/pull/13670/commits/4ac3cfecf470136723bb860c2363de50c176dcb7

**Review by av8ramit — COMMENTED** [review]
Remove @jhseu's changes.

**Comment by jhseu** [comment]
Yep, don't bring in my changes.

**Review by av8ramit — APPROVED** [review]

**Comment by case540** [comment]
Ok, I backed out the batch norm revert changes from the merge.
